src/components/Header.js

```diff
@@ -0,0 +1,32 @@
+import { useNavigate } from 'react-router-dom';
+import avatar from '../assets/avatar.svg';
+import './Header.css';
+
+function Header() {
+  const navigate = useNavigate();
+
+  const handleImageClick = () => {
+    navigate('/');
+  };
+
+  return (
+    <header className="Header">
+      <div className="header-container">
+        <div
+          className="logo"
+          onClick={handleImageClick}
+          aria-label="홈으로 이동"
+        ></div>
```

Comment:
P2:
해당 로고 클릭시 이동시키는 동작이니 아래처럼 하셔도 좋을 것 같습니다.
div 태그보다 a 태그가 더 시멘틱할 것 같아서요.

https://reactrouter.com/en/main/components/link

```suggestion
        <Link
          to='/'
          className="logo"
          aria-label="홈으로 이동"
        />
```